src/Sylius/Behat/Context/Api/Admin/ManagingAdministratorsContext.php

```diff
@@ -91,6 +91,14 @@ public function iSpecifyItsNameAs(string $username = null): void
         }
     }
 
+    /**
+     * @When I specify its :field as too long string
+     */
+    public function iSpecifyItsFieldAsTooLongString(string $field): void
+    {
+        $this->client->addRequestData(StringInflector::nameToCamelCase(lcfirst(trim(ucwords($field)))), str_repeat('a', 256));
```

Comment:
```suggestion
        $this->client->addRequestData(StringInflector::nameToCamelCase(lcfirst(trim(ucwords($field)))), str_repeat('a', self::MAX_(???)_LENGTH));
```

I have mixed feelings, as we have something generic and specific at the same time.